README.md

```diff
@@ -25,6 +25,10 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu
 
 **Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu/tree/master/springboot)]: Developed an online trading simulation platform using SpringBoot and microservice architecture. Designed the layer structure and Java components with MVC pattern and spring framework. Implemented Restful APIs to allow the customers to buy and sell stocks and remotely call the IEXCloud server to update stock market information. Coded integration and unit tests using Junit and Mockito libraries and applied those tests for Restful APIs. Programmed docker files to dockerize and deploy the back-end application and database separately. 
 
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu/tree/master/python_data_anlytics)]: Implemented a JupyterNotebook using Python libraries, including pandas, NumPy, Matplotlib. The project extracts useful information and visualizes them as histograms, line plots, boxplots, and tables in the notebook. The project analyzes data, answers specific business questions, and determines Market potential.
```

Comment:
You didn't implement the Jupyter notebook. 

You did data analytics on Sales data by using Python, Jupyter Notebook, Pandas, NumPy, etc..

You need to describe the purpose of the project.

you also need to mention the techniques you have used in data analytics

also need to talk about result